lib/api/web-element/commands/isSelected.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Determines if an element is selected.
+ *
+ * For more info on working with DOM elements in Nightwatch, refer to the <a href="https://nightwatchjs.org/guide/writing-tests/finding-interacting-with-dom-elements.html">Finding & interacting with DOM Elements</a> guide page.
+ *
+ * @example
+ * export default {
+ *   demoTest(browser: NightwatchAPI): void {
+ *     const result = browser.element('#main ul li a.first').isSelected();
+ *       .assert.valueEquals(true);
+ *   },
+ *
+ *   async demoTestAsync(browser: NightwatchAPI): Promise<void> {
+ *     const result = await browser.element('#main ul li a.first').isSelected();
+ *     console.log('element selected:', await result);
+ *   }
+ * }
+ *
+ * @since 3.0.0
+ * @method getText
+ * @memberof ScopedWebElement
+ * @instance
+ * @syntax browser.element(selector).isEnabled()
+ * @see https://www.w3.org/TR/webdriver#dfn-get-element-text
+ * @returns {ScopedValue<string>}
```

Comment:
These would also need to be changed accordingly.

Reply:
updated in [345b572](https://github.com/nightwatchjs/nightwatch/pull/4038/commits/345b572409c74688b489e1a208f3bfb2a75152cb)

Reply:
Thanks!